ci/utils.sh

```diff
@@ -54,3 +54,13 @@ architecture() {
             ;;
     esac
 }
+
+# 1 - if TRUE
```

Comment:
Having a shell function return `1` on success/true and `0` on failure/false is pretty unorthodox. Also there's a lot of unnecessary forking here. It doesn't really matter, it's not like it's performance-critical or anything, but (in the spirit of some of your other changes) it would probably be more idiomatic to do something like this:

```sh
case "${TARGET}" in
  i686-unknown-netbsd) return 1 ;;
  i686*|x86_64*)       return 0 ;;
esac
return 1
```

Then you can use it like:

```sh
if is_ssse3_target; then
  # do SSSE3 stuff
else
  # do not-SSSE3 stuff
fi
```

Reply:
> Having a shell function return 1 on success/true and 0 on failure/false is pretty unorthodox.

That's what I thought too but then decided to make it more traditional programming languages-like... :)
Good point though and I will change it.

Btw, I used `$?` because I tried to do:
```sh
is_ssse3_target() { return 0; }
test is_ssse3_target && echo "TRUE" || echo "FALSE"
```
And this would not work, but I see that this works:
```sh
if is_ssse3_target; then echo "TRUE"; else echo "FALSE"; fi
```
The alternative was to not use early-return and just `echo true/false`.